src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -114,10 +114,43 @@
             "$ref": "base.definition.schema.v1.json#/definitions/httpsUrl"
         },
         "taggable": {
-            "description": "A boolean flag indicating whether this resource type supports tagging.",
+            "description": "(deprecated) A boolean flag indicating whether this resource type supports tagging.",
             "type": "boolean",
             "default": true
         },
+        "tagging": {
+            "type": "object",
+            "properties": {
+                "taggable": {
```

Comment:
Do we need `taggable`? Or can we assume it's true if the top level `tagging` property is supplied. Also, I think it would make more sense for it to be called something like tagSettings or tagConfig

Reply:
yeah we may still need taggable since some resources need to mark it false to skip related contract tests in the future.

Reply:
we have thought about tagConfig before as well. But even when service team config e.g. "tagOnCreate" to be true, they still need to implement this feature in handlers. We won't help them do implementation based on those contents, so they are more like simply metadata reflecting what are supported for now.